PlayerTwo/playerGuessInput.js

```diff
@@ -116,7 +116,7 @@ const INCOMING_GUESS = "INCOMING GUESS: "; // put this at the beginning of the m
 
 guessConnection.onopen = () => { // when the connection to port 8080 is made
     console.log("player two connected"); // testing message
-    guessConnection.send("hello");
+    guessConnection.send(document.getElementById('guess-entry').value);
 }
```

Comment:
This was being hardcoded with the string "hello" instead of the actual guess value. Being changed like this will send the actual value that the player is trying to guess

Reply:
Yep, afraid that was a test message that was left in, thanks